### DISCUSSION TITLE
Get AWS infrastructure for the project

### DISCUSSION THREAD

**Discussion by jimmarino** [eclipse-edc/.github#20] [Planning]
Get AWS infrastructure for the project

**Comment by jimmarino** [comment]
@FlorianSeidelAWS said he could help with this

**Comment by FlorianSeidelAWS** [comment]
Hi all,

Policy prohibits me from giving access to externals to our internal accounts. I think the simplest way to get an AWS account for the project would be for me to provide AWS credits to pay for it, but that would require someone with a credit card to create the account. I’ll try to find someone at AWS who is willing to use their card for it. If someone else volunteers their credit card, that would speed things up. Again, credits would be provided to pay for the expenses.

Regards,

Florian Seidel
Global Solutions Architect
Automotive Vertical | Amazon Web Services
Email: ***@***.******@***.***>

Thoughts on our interaction? Provide feedback here<https://feedback.aws.amazon.com/?ea=flseidel&fn=Florian&ln=Seidel>.




From: Jim Marino ***@***.***>
Reply to: eclipse-dataspaceconnector/DataSpaceConnector ***@***.***>
Date: Friday, 3. September 2021 at 13:26
To: eclipse-dataspaceconnector/DataSpaceConnector ***@***.***>
Cc: "Seidel, Florian" ***@***.***>, Mention ***@***.***>
Subject: Re: [eclipse-dataspaceconnector/DataSpaceConnector] Get AWS infrastructure for the project (#55)


@FlorianSeidelAWS<https://github.com/FlorianSeidelAWS> said he could help with this

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/55#issuecomment-912466024>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUBBEIWEGWDCJAXOVYV76H3UACWD3ANCNFSM5DLRP5DQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.



Amazon Web Services EMEA SARL
38 avenue John F. Kennedy, L-1855 Luxembourg
Sitz der Gesellschaft: L-1855 Luxemburg
eingetragen im Luxemburgischen Handelsregister unter R.C.S. B186284

Amazon Web Services EMEA SARL, Niederlassung Deutschland
Marcel-Breuer-Str. 12, D-80807 Muenchen
Sitz der Zweigniederlassung: Muenchen
eingetragen im Handelsregister des Amtsgerichts Muenchen unter HRB 242240, USt-ID DE317013094
